src/main/features/dashboard/views/macro/claimStatus/claimFullAdmissionForDefendant.njk

```diff
@@ -34,21 +35,25 @@
       <p>{{ internalLink('Contact ' + claimantName, claimantDetailsPageURI) }}
         {{ t('if you need their payment details. Make sure you get receipts for any payments.') }}</p>
     {% elseif claim.response.paymentIntention.paymentOption === domain.PaymentOption.BY_SPECIFIED_DATE %}
-      <p>{{ t('You’ve offered to pay {{ claimantName }} {{ amount }} on {{ paymentDate }}.',
-          { claimantName: claimantName, amount: amount, paymentDate: paymentDate }) }}</p>
-      <p>{{ t('We’ll contact you when they respond.') }}</p>
+      <p>{{ t('You`ve said you owe {{ amount }} and offered to pay {{ claimantName }} by {{ paymentDate }}.',
```

Comment:
Use the correct apostrophe - not the backtick

Reply:
Fixed